src/app/app-routing.module.ts

```diff
@@ -1,25 +1,44 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { AddRecipePageComponent } from './components/add-recipe-page/add-recipe-page.component';
-import { RecipePageComponent } from './components/recipe-page/recipe-page.component';
-import { HomePageComponent } from './components/home-page/home-page.component';
 import { LoginComponent } from './components/login/login.component';
 import { AuthGuardService as AuthGuard } from './components/auth-guard.service';
 
 const routes: Routes = [
   {
     path: 'recipe/:id',
-    component: RecipePageComponent,
+    loadChildren: () =>
+      import('src/app/components/recipe-page/recipe-page.module').then(
+        (m) => m.RecipePageModule
```

Comment:
you don't have to put parentheses in this case `(m) => `, but I guess auto formatter did that

Reply:
Yes, auto formatter did that